### ISSUE TITLE
FR: astormetry.net Add server URL

### ISSUE THREAD

**Issue by dts350z** [astropy/astroquery#1545]
So you can target astormetry queries at local instances on the astrometry.net server, please consider adding the server URL so it can be localhost:8004 or whatever.

Thanks!

**Comment by dts350z** [comment]
Looks like maybe this is implemented but not yet documented?

I see code in astroquery/astrometry_net/__init__.py

but don't know how the config system for Astroquery works

**Comment by keflavich** [comment]
Yeah, the URL is meant to be configurable, but it's part of state and may only be documented at the top level rather than at an individual model level.  This is a reasonable use case, though, so maybe we should consider adding that documentation specifically for astrometry.net

cc @mwcraig
